unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -308,12 +309,46 @@ export default class ActionBinder {
         targetProduct: this.workflowCfg.productName,
         assetId: assetData.id,
       };
-      this.serviceHandler.postCallToService(
+      const finalizeJson = await this.serviceHandler.postCallToService(
         this.acrobatApiConfig.acrobatEndpoint.finalizeAsset,
         { body: JSON.stringify(finalAssetData) },
       );
+      if (Object.keys(finalizeJson).length !== 0) return false;
```

Comment:
any chance of `finalizeJson` being undefined?

Reply:
That could be true, have added a check for this.